app/src/main/res/navigation/nav_graph_2d_3d.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is a placeholder navigation?

Reply:
yes, for the future 2D and 3D models fragments, it is displayed on the homescreen main part